iceoryx_binding_c/test/moduletests/test_publisher.cpp

```diff
@@ -126,6 +128,28 @@ TEST_F(iox_pub_test, initPublisherWithNullptrForStorageReturnsNullptr)
     EXPECT_EQ(iox_pub_init(nullptr, "all", "glory", "hypnotoad", &options), nullptr);
 }
 
+// this crashes if the fixture is used, therefore a test without a fixture
+TEST(iox_pub_test_DeathTest, initPublisherWithNotInitializedPublisherOptionsTerminates)
+{
+    iox_pub_options_t options;
+    iox_pub_storage_t storage;
+
+    EXPECT_DEATH({ iox_pub_init(&storage, "a", "b", "c", &options); }, ".*");
+}
+
+TEST_F(iox_pub_test, initPublisherWithDefaultOptionsWorks)
+{
+    iox::roudi::RouDiEnvironment roudiEnv;
+
+    iox_runtime_init("hypnotoad");
```

Comment:
could you please create a test fixture and move the runtime_init in there. also the publisher creation can be stored there to avoid code duplication

Reply:
No, not in this case. The RouDiEnvironment is only used for a few tests